.github/workflows/ci.yml

```
@@ -17,11 +17,59 @@ jobs:
+          git diff --exit-code lib/l10n || {
+            echo "::error::lib/l10n not up-to-date. Run 'flutter gen-l10n' locally and commit generated files.";
+            git --no-pager diff -- lib/l10n || true;
+            exit 1;
+          }
       - run: flutter analyze
       - run: flutter pub run dart_code_metrics:metrics analyze lib --disable-sunset-warning || true
       - run: flutter test --no-pub
 
+  log-consent-contract:
+    name: Edge Contract / log_consent
+    if: ${{ secrets.SUPABASE_URL != '' && secrets.SUPABASE_ANON_KEY != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)